bigbluebutton-html5/imports/ui/stylesheets/variables/typography.scss

```diff
@@ -4,7 +4,7 @@ $font-family-monospace:   Menlo, Monaco, Consolas, "Courier New", monospace !def
 
 $font-family-base:        $font-family-sans-serif !default;
 
-$font-size-base:          1rem !default;
+$font-size-base:          0.9rem !default;
```

Comment:
Does this font size complains with the WCAG 2.0 AA?
Changing this value will impact in all components, not only the chat input as requested in #5980.